bin/otomi

```diff
@@ -68,7 +68,9 @@ uname -a | grep -i darwin >/dev/null && helm_config="$HOME/Library/Preferences/h
 function show_usage() {
   echo "otomi usage:
     bash                run interactive bash shell in otomi-core container
+    bats                runs BATS on bin/tests (see docs/BATS.md)
```

Comment:
You can agree whether we want this in our CLI or not. But this is a damn easy way to execute tests. The command is accessible to everyone and the whole setup is available through `otomi`.

Reply:
See if `otomi x bats` could be easier to not expose developer tooling.

Reply:
can we just use `otomi x bats ...` to achieve the same? if so then don't expose dev stuff please